SConstruct

```diff
@@ -144,6 +144,7 @@ elif os.name == "nt" and methods.get_cmdline_bool("use_mingw", False):
 # want to have to pull in manually.
 # Then we prepend PATH to make it take precedence, while preserving SCons' own entries.
 env = Environment(tools=custom_tools)
+env.Decider("MD5-timestamp")
```

Comment:
It's minor, but I'd like to keep the SCons speed related options together, close to where they're documented. So I'd keep this around line 510, but before the check for fast_unsafe (and maybe tweak the comment to reflect current understanding of what is and is not safe).

Reply:
I made some adjustments to the placement, and updated the comment